src/Microsoft.Windows.CsWin32/templates/BOOL.cs

```diff
@@ -1,6 +1,11 @@
 ﻿partial struct BOOL
 {
+	public const int Size = sizeof(int);
+	public static BOOL TRUE { get; } = new(true);
+	public static BOOL FALSE { get; } = new(false);
 	internal BOOL(bool value) => this.Value = value ? 1 : 0;
 	public static implicit operator bool(BOOL value) => value.Value != 0;
 	public static implicit operator BOOL(bool value) => new BOOL(value);
+	public static bool operator true(BOOL value) => value.Value != 0;
+	public static bool operator false(BOOL value) => value.Value == 0;
```

Comment:
What do these operator overloads add, considering we already define an implicit `bool` conversion operator?

Reply:
I asked chat gpt as I have no idea. There is a difference apparently.

> The true and false operators define how the BOOL type is directly evaluated in boolean expressions. They don't actually convert the type to bool; they just inform the compiler how to treat BOOL instances in boolean contexts.
> 
> The implicit conversion operator actually converts the BOOL type to a bool. This means wherever a bool is expected (e.g., in assignments, method parameters), a BOOL instance can be provided, and it will be automatically converted to bool according to the logic defined in the operator.

Reply:
I did read up on Bing Chat and the doc you mentioned (and others), but none of them convinced me that this makes any difference. There may be one, but I'm not going to add API without knowing its impact.